docs/guides/managing-existing-services.md

```diff
@@ -64,7 +64,7 @@ for Production.
 For example:
 
 ```
-curl -XPUT https://harmony.uat.earthdat.nasa.gov/service-image-tag/#canonical-service-name  -H 'Authorization: bearer <your bearer token>'  -d '{"tag": "new-version"}' -H 'Content-type: application/json'
+curl -XPUT https://harmony.uat.earthdata.nasa.gov/service-image-tag/#canonical-service-name  -H 'Authorization: bearer <your bearer token>'  -d '{"tag": "new-version"}' -H 'Content-type: application/json'
```

Comment:
We just had some discussion as to whether or not `bearer` needed to be capitalized. I assume you found this typo when you were testing. Did lowercase `bearer` work? If not, can you also fix that with this update?

Reply:
When I did it, Bearer was capitalized. so I can't help there